js/scripts.js

```diff
@@ -1,8 +1,8 @@
-AWS.config.update({
+/*AWS.config.update({
     region: 'eu-west-1',
     accessKeyId: 'fakeMyKeyId', // add this as env var
     secretAccessKey: 'fakeSecretAccessKey' // add this as env var
-});
+});*/
```

Comment:
Could you please remove this commented out code since it's no longer needed?

Reply:
Resolved.